base/docs/bootstrap.jl

```diff
@@ -4,7 +4,15 @@ module DocBootstrap
 
 export @doc
 
-const docs = []
+type List
+    head
+end
+type Node
+    data
+    next
+end
```

Comment:
Yeah, I think you're just going from `pointer -> Any` to `pointer -> pointer -> Any`, so there's no real benefit here.